### PR TITLE
gettinginputs

### PR DESCRIPTION
**Pull request by vennela413**

### CODE REVIEW

File: forms.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+              <input type="checkbox" name="checkbox" value="I agree Terms and Conditions">
+              I agree to the <a href="#">terms and conditions</a>
+            </label>
+          </div>
+         </div>
+        </div>
+       </div>
+     <br><br>
+     <button class="button is-success" id="submit">Login</button>
+     <button class="button is-warning" id="button" onclick="myfunction()" type="reset">reset</button>
```
Comment (su66uu) [inline forms.html]: You are still using Javascript events. Use jquery events.

File: form.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      },
+      member:{
+        required:true,
+      },
+      checkbox:{
+        required:true,
+      }
+    }
+  });
+  function myfunction(){
```
Comment (su66uu) [inline form.js]: The name `myFunction` doesn't make any sense to me

**Review by su66uu — CHANGES_REQUESTED** [review]

File: forms.html (hunk 1 above)
Comment (vennela413) [reply forms.html]: ok anna

File: form.js (hunk 2 above)
Comment (vennela413) [reply form.js]: i change that name

**Comment by su66uu** [comment]
Seems everything looks good except the alignment. Are you not using any formatter? If not check this out https://scotch.io/tutorials/code-formatting-with-prettier-in-visual-studio-code

You can format your code using prettier extension

File: forms.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.8.2/css/bulma.css"
+    />
+    <link rel="stylesheet" href="form.css" />
+    <script src="http://code.jquery.com/jquery-3.5.1.js"></script>
```
Comment (su66uu) [inline forms.html:9]: Don't keep commented code in the PRs. Please remove it

**Review by su66uu — CHANGES_REQUESTED** [review]

File: forms.html (hunk 3 above)
Comment (vennela413) [reply forms.html:9]: ok anna

**Review by su66uu — APPROVED** [review]

**Review by su66uu — APPROVED** [review]

**Review by su66uu — APPROVED** [review]

**Pull request merged**